InternTrack.Core.Api.Infrastructure.Provision/Brokers/Clouds/CloudBroker.Plans.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using Microsoft.Azure.Management.AppService.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+
+namespace InternTrack.Core.Api.Infrastructure.Provision.Brokers.Clouds
+{
+    public partial class CloudBroker
+    {
+        public async ValueTask<IAppServicePlan> CreatePlanAsync(string planName, IResourceGroup resourceGroup)
+        {
+            return await azure.AppServices.AppServicePlans
+                .Define(planName)
+                .WithRegion(Region.USWest3)
+                .WithExistingResourceGroup(resourceGroup)
```

Comment:
We need to pass in here a connection to the database

Reply:
where in the chain do I pass it in? I couldn't find the option for it

Reply:
@hassanhabib

Reply:
https://github.com/microsoft/mixedreality.dmx.core/tree/main/DMX.Core.Api.Infrastructure.Provision

@Pwongchaiya this might be a good example

Reply:
@hassanhabib I'm not seeing a difference between "DMX.Core CloudBroker.Plans" vs "InternTrack.Core CloudBroker.Plans". I'm very confused as what to do.

Reply:
@Pwongchaiya the plan isn't where a db connection string goes.
WebApp is where it goes. Check the files there.

Reply:
@hassanhabib db connection string is already in our ColudBroker.WebApp.cs 
![Pwongchaiya_InternTrack Core](https://user-images.githubusercontent.com/68671925/234078824-9e5d3078-e6f7-4df2-9ba9-11a9570ce6ab.png)